mlearning/libcmsisdsp/Makefile

```diff
@@ -0,0 +1,566 @@
+############################################################################
+# apps/mlearning/libcmsisdsp/Makefile
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+include $(APPDIR)/Make.defs
+
+ifeq ($(CONFIG_EXTERNALS_CMSIS_DSP_ARM_MATH_MATRIX_CHECK),y)
+CFLAGS += -DARM_MATH_MATRIX_CHECK
```

Comment:
The flag add here isn't enough to get the full optimized code:
https://github.com/ARM-software/CMSIS_5/blob/develop/CMSIS/DSP/config.cmake
https://github.com/ARM-software/CMSIS_5/blob/develop/CMSIS/DSP/cmsisdspconfig.py
You need add the same flag as the original cmake script.

Reply:
Ok.